server/api/users/users.test.ts

```diff
@@ -57,4 +60,88 @@ describe('auth router', () => {
       });
     });
   });
+
+  describe('PATCH /:id', () => {
+    it('returns 403 if profile id is not equal to logged in user id', async () => {
+      const res = await server.exec.patch('/api/users/54321');
+      expect(res.status).toBe(403);
+    });
+
+    it('returns 400 if request body is empty object', async () => {
+      const user = await User.create({
```

Comment:
We recently merged #48 that includes a `createUser()` method for randomly generating a user that can be used here instead of manually typing it out.

You can see an example of how the method is used for the `'GET /:id'` test further up

Reply:
That's great, thanks for the heads up! I've gone ahead and replaced the user creation code with it.